doc/source/conf.py

```diff
@@ -52,14 +52,16 @@
               'numpydoc', # used to parse numpy-style docstrings for autodoc
               'ipython_sphinxext.ipython_directive',
               'ipython_sphinxext.ipython_console_highlighting',
+              'IPython.sphinxext.ipython_console_highlighting',  # lowercase didn't work
```

Comment:
Note there is a difference here in `_` vs `.`. The one already present with the `_` is our vendored version (and the directory is with lower case, so that works). What was the reason you needed this? 
(although we should maybe migrate to just use IPython itself instead of our own version, but should look again at why we exactly were doing this)